src/applications/rx/containers/Active.jsx

```diff
@@ -270,7 +270,7 @@ class Active extends React.Component {
 
   render() {
     return (
-      <ScrollElement id="rx-active" name="active" className="va-tab-content">
```

Comment:
Is this needed for your changes or unrelated cleanup?

Reply:
Yup, these changes were necessary for visual effect. Without moving the `va-tab-content` class one level up, the line under the Rx tabs would have been under the MHV moving notice instead of above and connected with the tabs.